rh6/pbuild/config.py

```diff
@@ -484,7 +484,7 @@ def InitializeSSH(self):
             process = subprocess.Popen(
                 [
                     'ssh', '-A', host,
-                    'grep github.com, ~/.ssh/known_hosts > /dev/null 2> /dev/null || ssh -o StrictHostKeyChecking=no -o HashKnownHosts=no -T git@github.com; grep github.com, ~/.ssh/known_hosts > /dev/null 2> /dev/null || ssh -o StrictHostKeyChecking=no -T git@github.com'
+                    'grep github.com, ~/.ssh/known_hosts > /dev/null 2> /dev/null || ssh -o StrictHostKeyChecking=no -o HashKnownHosts=no -T git@github.com; grep github.com, ~/.ssh/known_hosts > /dev/null 2> /dev/null || ssh -o StrictHostKeyChecking=no -T git@github.com; mkdir -p /root/rpmbuild; mkdir -p /root/rpmbuild/SPECS; mkdir -p /root/rpmbuild/SOURCES; mkdir -p /root/rpmbuild/SRPMS; mkdir -p /root/rpmbuild/RPMS;'
```

Comment:
Support the directory creation in rh6/SPECS/buildrpms.sh file. This script is executed for rpm creation and it will be good to have all modification here.

Reply:
Same as above